clang/lib/Driver/ToolChains/SYCL.h

```diff
@@ -31,9 +31,10 @@ void TranslateSYCLLinkerArgs(Compilation &C, const llvm::opt::ArgList &Args,
                              llvm::opt::ArgStringList &CmdArgs);
 
 void constructLLVMForeachCommand(Compilation &C, const JobAction &JA,
-                                 std::unique_ptr<Command> InputCommand,
-                                 llvm::opt::ArgStringList &ForeachArgs,
-                                 const InputInfo &Output, const Tool *T);
+                                       std::unique_ptr<Command> InputCommand,
+                                       const InputInfoList &InputFiles,
+                                       const InputInfo &Output, const Tool *T,
+                                       StringRef Ext);
```

Comment:
Is clang-format OK with this? (Just asking)

Reply:
No, I forgot run it lat time...

Reply:
Should be fixed now.